### ISSUE TITLE
Application health check

### ISSUE THREAD

**Issue by REBELinBLUE** [REBELinBLUE/deployer#11]
After a deploy (and maybe periodically) a HTTP request should be made to the URL to ensure it is up

Maybe expand the testing more so allow for multiple URLs?

**Comment by iflamed** [comment]
If run this health check periodically, so what is differences with the heartbeats flow?
Should we just support it run only after a deploy?

**Comment by REBELinBLUE** [comment]
This was a request from someone at work, I think the idea was basically to periodically call the project URL (and on deploy) and ensure it is returning 200 OK. Not really sure about it yet

**Comment by iflamed** [comment]
Ok, I will make a url checklist for this. But only check the response body and status code.

**Comment by iflamed** [comment]
@REBELinBLUE  There is code style error like below. How do i fix it? Really need your help.
```shell
/www/sites/deployer/app/Project.php:13	The class Project has a coupling between objects value of 13. Consider to reduce the number of dependencies under 13.
```

**Comment by REBELinBLUE** [comment]
That is very odd that PHPMD complains about the relationships like that

**Comment by iflamed** [comment]
@REBELinBLUE  I received a email, you ask me to create a new pull request. But i see you have been merged my branch's code change.
